crud-frontend/src/components/login/login.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Why call this `login/login.tsx` rather than `login/index.tsx` or simply `login.tsx`?

Also I believe the convention is to use capital letters for the file name, `Login.tsx`.